tensorboard/webapp/widgets/histogram/histogram_component.ts

```diff
@@ -260,6 +260,17 @@ export class HistogramComponent implements AfterViewInit, OnChanges, OnDestroy {
       : '';
   }
 
+  updateColorOnHover(event: MouseEvent, isHover: boolean) {
+    if (!this.isLinkedTimeEnabled(this.linkedTime)) {
```

Comment:
Maybe comment that when link time is disabled all histogram lines are colored so there is no need to update?